packages/firestore/CONTRIBUTING.md

```diff
@@ -2,10 +2,31 @@
 
 See [Contributing](../../CONTRIBUTING.md) for general information on
 contributing to the Firebase JavaScript SDK (including Cloud Firestore).
+Follow instructions there to install dependencies and set up the testing
+environment.
+
+## Setting up an account for testing
+
+You will need a production project to test the Firestore SDK. You can create
+a new project by visiting the 
+[Firebase Console](https://console.firebase.google.com/). Make sure that the 
+project has Cloud Firestore enabled in the database section of the console.
+
+While setting up tests, you can choose the project manually, or specify the 
+project directly with the following command:
```

Comment:
We should call out that it needs to be done at the root of the SDK, not within the firestore folder.

Reply:
Or better yet, can you just point to the Automated Setup section in the main README?

Reply:
I think a better of of thinking about this is that we should link out to the main README for one-time setup, but keep instructions here for how to work with Firestore, including over time.

For example, when doing a `git pull` to come up to date with changes others have made, do we need to `yarn build` at the root?